src/features/Dashboard/components/RecentTransactionList.tsx

```diff
@@ -227,19 +226,32 @@ export default function RecentTransactionList({ accountsList, className = '' }:
               address={receiverAddress || to}
             />
           ),
-          <Amount
-            key={3}
-            assetValue={`${bigify(amount).toFixed(4)} ${asset.ticker}`}
-            fiat={{
-              symbol: getFiat(settings).symbol,
-              ticker: getFiat(settings).ticker,
-              amount: convertToFiat(amount, getAssetRate(asset)).toFixed(2)
-            }}
-          />,
+          <Box key={3}>
+            {isMobile ? (
+              <Amount
+                alignLeft={true}
```

Comment:
Can we just do `alignLeft={isMobile}` to reduce duplication?

Reply:
Good Idea!